doc/content/tutorial/01-getting-started/index.rst

```diff
@@ -3,18 +3,42 @@
 1. Getting Started
 ==================
 
+Introduction
+------------
+
+Lona is a full-stack web-application framework. That means it is possible to
+write a whole, self-contained, web application in pure Python. Lona runs
+completely on the server. Traditionally that would mean that Lona could only do
+"backend" tasks, like talking to a database, but no "frontend" tasks like
+responding to a click event in the browser. Lona implements a generic frontend
+that can be controlled by the backend, live,
+over websockets.
```

Comment:
Great intro :) You might want to join line 14 and 15

Reply:
Thanks! :)
Yes, you are right. I joint both lines